lib/alias.rb

```diff
@@ -40,6 +40,10 @@ def valid_symlink?
         false
       end
 
+      def link()
+        FileUtils.ln_sf script, symlink
```

Comment:
Just a thought: maybe we should just use `FileUtils.ln_s` here and only delete if it appropriate e.g. an existing symlink?

Reply:
Agreed. Updated.